cps/CP61.md

```diff
@@ -5,8 +5,8 @@ authors: ImdioR
 contributors: Cassidy, Luis
 uses-component: CP4 technical
 date-proposed: 2023-08-16
-date-ended: 
-status: polling
+date-ended: 2023-08-28
+status: passed
 ---
 
 ## Short Summary

```

Comment:
Since we don`t need Opensquare snapshot vote for on-chain voting, i guess will make sense to include instead of Opensquare link:
Link to the on-chain vote: https://centrifuge.subscan.io/council/70

Reply:
Good poing, and I was actually thinking about that yesterday. I will add link to on-chain vote for those CPs where it is relevant.

Link to SubSquare or Subscan? I personally prefer SubSquare as it is a more clean UI and contains the same info. WDYT?